src/filters/write_vtt.c

```diff
@@ -247,6 +257,7 @@ GF_Err vttmx_process(GF_Filter *filter)
 		return GF_OK;
 	}
 	data = (char *) gf_filter_pck_get_data(pck, &pck_size);
+	ctx->first = GF_FALSE;
```

Comment:
we should have on option on vttmx to enable this feature, for cases where we want to dump exactly what is in the stream.

I'd suggest replacing `first` by `noempty` and exporting it as an option in WebVTTMxArgs